schema/ga4gh.yaml

```diff
@@ -23,7 +23,7 @@ identifiers:
     Allele: VA
     VariationSet: VS
     Text: VT
```

Comment:
@ahwagner Here's a whopper.  I think we should consider "Text" or "VariationText" to be a universal and ambigous thing that can be used for any "text string" not just for variation. This would allow us to using it for "Conditions", "Drugs" and other value-object-like domain entities that we will be using.  Just because it's under the VRS umbrella does not mean it is specialized to be a Variation (I think).  Unless we think that subclassing these utilitye "text" concepts to domain specific types (i.e. ConditionText, DrugText, etc...) is important for folks to know that it meant to be a Condition, Drug, etc... .  Just thinking that if we wanted to do this then we would change the prefix to "TX" or "TXT" and modify the documentation to emphasize that it is truly a utility class that open to interpretation and will require the producer and consumer to "imply and infer" it's meaning (which is what they do anyway).

Reply:
this is most definitely a "Draft" data type currently, but is something I will be implementing in ClinVar for Variation and Conditions.